### PR TITLE
Baris/eng 293 remove git init

### PR DESCRIPTION
**Pull request by bcdurak**
## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Describe changes
the instructions to do `git init` while creating a ZenML repository have been removed from the examples and the docs

### CODE REVIEW

**Review by alex-zenml — APPROVED** [review]
💪

File: src/zenml/integrations/integration.py [hunk 1]
```diff
@@ -50,13 +49,17 @@ class Integration(metaclass=IntegrationMeta):
     def check_installation(cls) -> bool:
         """Method to check whether the required packages are installed"""
         try:
-            for req, command in cls.SYSTEM_REQUIREMENTS.items():
+            for requirement, command in cls.SYSTEM_REQUIREMENTS.items():
```
Comment (htahir1) [inline src/zenml/integrations/integration.py:52]: I think @schustmi already has a branch for this :-)

File: src/zenml/cli/__init__.py [hunk 2]
```diff
@@ -52,16 +52,8 @@
...
-
-.. code:: bash
-
-   git init
-
-Once your directory is setup with a (usually hidden) ``.git`` folder,
-initialize the directory with ZenML’s own config and resource management
-tools. Type:
+When you begin a project, initialize the directory with ZenML’s
+own config and resource management tools. Type:
```
Comment (htahir1) [inline src/zenml/cli/__init__.py:56]: Isnt this missing something?

**Review by htahir1 — COMMENTED** [review]
Nice one! Just a few suggestions. Also, I think was  a mention in the docs somewhere that we were building on top of git, maybe we should also remove that if it exists

File: src/zenml/integrations/integration.py (hunk 1 above)
Comment (bcdurak) [reply src/zenml/integrations/integration.py:52]: I merged @schustmi 's branch before the request so it should be good to go.

File: src/zenml/cli/__init__.py (hunk 2 above)
Comment (bcdurak) [reply src/zenml/cli/__init__.py]: I have changed it a bit more to reflect the current situation.

**Comment by bcdurak** [comment]
@htahir1 Thanks for the tip regarding the docs. I have also changed the corresponding section. I hope it is a bit clearer now.

**Review by htahir1 — APPROVED** [review]

**Pull request merged**